run_unittests.py

```diff
@@ -739,6 +742,47 @@ def test_libdir_must_be_inside_prefix(self):
         self.init(testdir)
         self.assertRaises(subprocess.CalledProcessError, self.setconf, '-Dlibdir=/opt', False)
 
+    def test_prefix_dependent_defaults(self):
+        '''
+        Tests that configured directory paths are set to prefix dependent
+        defaults.
+        '''
+        testdir = os.path.join(self.common_test_dir, '1 trivial')
+        expected = {
+            '/opt': {'prefix':'/opt',
+                    'bindir':'bin', 'datadir':'share', 'includedir':'include',
+                    'infodir':'share/info', 'libdir':'lib',
+                    'libexecdir':'libexec', 'localedir':'share/locale',
+                    'localstatedir':'var', 'mandir':'share/man',
+                    'sbindir':'sbin', 'sharedstatedir':'com',
+                    'sysconfdir':'etc'},
+            '/usr': {'prefix':'/usr',
+                    'bindir':'bin', 'datadir':'share', 'includedir':'include',
+                    'infodir':'share/info', 'libdir':'lib',
+                    'libexecdir':'libexec', 'localedir':'share/locale',
+                    'localstatedir':'/var', 'mandir':'share/man',
+                    'sbindir':'sbin', 'sharedstatedir':'/var/lib',
+                    'sysconfdir':'/etc'},
+            '/usr/local': {'prefix':'/usr/local',
+                    'bindir':'bin', 'datadir':'share', 'includedir':'include',
+                    'infodir':'share/info', 'libdir':'lib',
+                    'libexecdir':'libexec', 'localedir':'share/locale',
+                    'localstatedir':'/var/local', 'mandir':'share/man',
+                    'sbindir':'sbin', 'sharedstatedir':'/var/local/lib',
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [E231] missing whitespace after ':'

<!-- Comment by [SideCI](https://sideci.com) -->